INSTALLATION.md

```diff
@@ -1,87 +1,64 @@
-2. Once downloaded, double clik on the file
-> **_NOTE:_** You may need to authorise the download of this file as your Mac won't recognise it. You can authorise the installer to run by right clicking on the installer and click on `Open` from Finder. A dialog box will open asking you if you are sure, press the `Open` button and the installer should start.
-3. A terminal should open automatically where you can follow the steps of the installation. Just watch it working, and at some point you'll need to add your password.
-* Why do you need to enter your password? Simply because Thyra is an application and to add this application to your Mac you need to validate that you wish to install this app; like you do for app from the app store. 
-4. Once the installation is finished, you will see a new icon on the top nav bar (see screenshot below). You can now close your terminal (if not done already) and start browsing the blockchain and all our applications available.
-<img width="177" alt="image" src="https://user-images.githubusercontent.com/109611779/223798813-92a16141-19f6-415b-b9d8-2554eb814edc.png">
+## Debian Linux Installation
 
-> **_NOTE:_** If you quit Thyra, the icon tray will disappear. Normal! If you want to add it back, open your finder and look for "thyra" in all your mac. You should see the following file "XXXX"; click on it and the app will be back on your top nav bar. 
+### GUI Installation
 
-> **_TROUBLESHOOTING:_** Do not hesitate to report any problem or request help by [opening an issue](https://github.com/massalabs/thyra/issues/new).
+1. Download the latest version of **MassaStation** installer for Debian Linux from [here](https://github.com/massalabs/thyra/releases/latest/).
+2. Open your file manager and navigate to the location where the `.deb` package is saved.
+3. Right-click on the `.deb` package and choose "Open with Software Install" or a similar option.
+4. The package manager will launch and display **MassaStation** installation page.
+5. Review the package information and dependencies, if any, and click on the "Install" button.
+6. If prompted, enter your administrator password to authorize the installation.
+7. The installation will commence, and you will see a progress bar indicating the status.
+8. Once the installation is complete, you will receive a notification confirming the successful installation.
 
-🎉 Congratulations!! Your gate to Massa blockchain is now installed and you can start your massalian journey.
-You can now browse the **websites on-chain** seamlessly. If you need to take the pressure off, maybe a little [game](https://flappy.massa) can help.
-If you want to get down to business, you can start your [Massalian journey](https://my.massa/thyra/plugin/massalabs/wallet) right away!
+### Terminal Installation using apt
 
+1. Open a terminal on your Debian Linux system.
```

Comment:
this way of installing doesn't contain the stop of downloading the file. I would either add it here, or move it before title GUI Installation